src/blinkstick/configs.py

```diff
@@ -0,0 +1,31 @@
+from blinkstick.enums import BlinkStickVariant
+from blinkstick.models import Configuration
+
+_VARIANT_CONFIGS: dict[BlinkStickVariant, Configuration] = {
+    BlinkStickVariant.BLINKSTICK: Configuration(
+        mode_change_support=False,
+    ),
+    BlinkStickVariant.BLINKSTICK_PRO: Configuration(
+        mode_change_support=True,
+    ),
+    BlinkStickVariant.BLINKSTICK_NANO: Configuration(
+        mode_change_support=True,
+    ),
+    BlinkStickVariant.BLINKSTICK_SQUARE: Configuration(
+        mode_change_support=True,
+    ),
+    BlinkStickVariant.BLINKSTICK_STRIP: Configuration(
+        mode_change_support=True,
+    ),
+    BlinkStickVariant.BLINKSTICK_FLEX: Configuration(
+        mode_change_support=True,
+    ),
+    BlinkStickVariant.UNKNOWN: Configuration(
+        mode_change_support=False,
+    ),
+}
```

Comment:
@arvydas Is this correct? It wasn't clear to me if the non-pro multi addressable LED models (e.g. square, strip etc.) also supported mode change.. I figure they can't do mode 0 for a single RGB LED, but I wasn't clear on whether mode 1 (inverse mode) was supported in addressable mode.